### ISSUE TITLE
Log message to wrong log level 

### ISSUE THREAD

**Issue by dustymabe** [projectatomic/atomicapp#659]
When running the wordpress example on the docker provider you see this:

```
2016-03-31 21:46:18,075 - [INFO] - docker.py - WARNING: Using --name provided within artifact file.
```

Looks like the log statement needs to get fixed because it is printing a warning to the [INFO] level.

**Comment by dustymabe** [comment]
@shiminsh,  @abhi1004, @nurav 

Would one of you like to take a stab at it. This one should be easy.

**Comment by nurav** [comment]
I'd like to! I'll issue a PR soon.

**Comment by shiminsh** [comment]
@nurav you're working on it or I can start working on this.

--
Kumari Shalini

**Comment by nurav** [comment]
I'm working on it.

**Comment by abhi1004** [comment]
@nurav are you still working on this?

**Comment by dustymabe** [comment]
Fixed in #759
